script/main.js

```diff
@@ -0,0 +1,52 @@
+
+function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+  value : 1,
+  roll: function(){
+    this.value = getRandomIntInclusive(1,6);
+    return this.value;
+  }
+}
+
+const diceTwo = {
+  value : 1,
+  roll: function(){
+    this.value = getRandomIntInclusive(1,6);
+    return this.value;
+  }
+}
+
+
+let button = document.querySelector('button').addEventListener('click', event => {
```

Comment:
There's no need to save the button in a variable if you are not going to reuse it. The better way to do this is:

```javascript
document.querySelector('button').addEventListener('click', event => {
   // Your code goes here....  
});
```